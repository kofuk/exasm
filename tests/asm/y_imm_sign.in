addi r3, -0
addi r3, -0x0
addi r3, -7
addi r3, -0xb
