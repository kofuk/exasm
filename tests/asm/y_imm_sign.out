@00 00100011 00000000 // addi r3, 0x00
@02 00100011 00000000 // addi r3, 0x00
@04 00100011 11111001 // addi r3, -0x07
@06 00100011 11110101 // addi r3, -0x0B
